README.md

```diff
@@ -62,6 +62,20 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 ### Disabled by default
 
+The perf collector may not work by default on all Linux systems due to kernel
+configuration and security settings. To allow access, set the following sysctl parameter:
+
+```
+sysctl -w kernel.perf_event_paranoid=X
```

Comment:
What value should we recommend for most useful data? Based on the docs, I would say `0` is the right option.

Reply:
Yeah, I think that makes sense.

Reply:
I think we should add a few words on the security implications of this. The only CVEs I found are about bug in the perf subsystem itself but I would assume this is also about timing/side channel attacks?

Reply:
I added a few notes on it, I think that makes a lot of sense as well.

Reply:
I would like to not include `CAP_SYS_ADMIN` as even a not recommended option. We very much want to keep users from running the node_exporter as root. Golang currently has no ability to drop privileges, so it's always a vector for escalation.

Reply:
Sounds good to me!

Reply:
I'm not sure if it makes sense but the [Syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller) is also a good resource for kernel related bugs found via fuzzing.